common/state_edicts/03_protect_trade.txt

```diff
@@ -40,46 +40,80 @@ edict_protect_trade = {
 		province_trade_power_value = 1.0
 	}
 	color = {  113 11 43 }
+
 	ai_will_do = {
-	
-		# trade_income_percentage = 0.10
-	
-		factor = 30
+		factor = 5
```

Comment:
Was AI enabling protect trade too much?

Reply:
Older value was overwriting any economic consideration. And there was no modifier to stop AI from using edicts.